vaadin-grid-flow-parent/vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/Grid.java

```diff
@@ -2929,6 +2929,8 @@ public void removeColumn(Column<T> column) {
         Objects.requireNonNull(column, "column should not be null");
 
         ensureOwner(column);
+        List<GridSortOrder<T>> order = new ArrayList<>();
+        setSortOrder(order, false);
```

Comment:
Wouldn't this reset all sorting from the Grid whenever a single column gets removed?

Reply:
Yes, and I think it is the correct behavior.